pkgs/applications/misc/kiwix/default.nix

```diff
@@ -28,6 +28,7 @@ let
               then { tar = xulrunner64_tar; sdk = xulrunnersdk64_tar; }
               else { tar = xulrunner32_tar; sdk = xulrunnersdk32_tar; };
 
+  ctpp2 = callPackage ../../../development/libraries/ctpp2 { inherit stdenv; };
```

Comment:
Sorry I do not understand the purpose of using `callPackage` here. How is stdenv in this context different?